packages/multitest/Cargo.toml

```diff
@@ -14,14 +14,14 @@ crate-type = ["cdylib", "rlib"]
 [features]
 
 [dependencies]
-nois-drand = { path = "../../contracts/nois-drand"}
-nois-gateway = { path = "../../contracts/nois-gateway"}
-nois-icecube = { path = "../../contracts/nois-icecube"}
+nois-drand = { path = "../../contracts/nois-drand" }
+nois-gateway = { path = "../../contracts/nois-gateway" }
+nois-icecube = { path = "../../contracts/nois-icecube" }
```

Comment:
Do you have a editor plugin that does those formatting changes automatically? Looks nice

Reply:
It is CLion with Rust plugin or just RustRover both from JetBrains.